src/www/components/landingpage/LandingPage.scss

```
@@ -0,0 +1,612 @@
+.container {
+  background-image: linear-gradient(  rgb(129, 129, 129) ,white);
+  height: 325vh; //parallax(110vh) + 2 pages(200vh) + 15vh more empty space
+  max-width: 100vw;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Times New Roman', Times, serif;
+
+  .thirdpage{
+    
+    .phrase{
+      margin-top: 20vh;
+      margin-bottom: 10vh;
+      float: left;
+      margin-left: 1  5vw;
+      font-size: calc(3.5vh + 1vw);
+    }
+
+    .parallaxbg{
+      
+      width: 74vw;
+      height: 65vh;
+      margin-left: 13vw;
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      border-radius: 0.5rem;
+      
+      
+      .panel{
+        width: 80vw;
+        height: 65vh;   //same as parallaxbg height 
+        flex-wrap: wrap;
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        margin-left: 1rem;
+        margin-right: 1rem;
+        box-shadow: inset 0px 0px 30px 1px black;
+
+        .card{
+          margin-right: 2rem;
+          height: 23vh;
+          width: 18vw;
+          min-width: 13rem;
+          min-height: 6.5rem;
+          align-items: center;
+          border-radius: 1rem;
+          background-color: rgba($color: white, $alpha: 0.7);
+          margin-left: 2rem;
+          min-width: 13rem;
+
+          .box{
+            align-items: center;
+            display: flex;
+            flex-direction: row;
+            //justify-content: center;
+            border-bottom: solid;
+
+            .category{
+              margin-top: 1vh;
+              margin-left: 1vh;
+              margin-bottom: 1vh;
+              font-size: calc(1vh + 1vw);
+            }
+
+            .boximg{
+              margin-top: 1.5vh;
+              margin-left: 1.5vh;
+              margin-bottom: 1.5vh;
+              height: calc(4vh + 1vw);
+              width: calc(4vh + 1vw);
+              border: solid;
+              border-width: 2px;
+              border-color: rgba($color: black, $alpha: 0.5);
+            }
+          }
+
+          .boxdesc{
+            margin-top: 1vh;
+            margin-left: 1vh;
+            margin-right: 1vh;
```

I think we need a bottom margin for this one too, so replace these lines with
```suggestion
            margin: 1vh;
```